src/main/kotlin/frc/robot/subsystems/climb/ClimbIO.kt

```diff
@@ -0,0 +1,29 @@
+package frc.robot.subsystems.climb
+
+import edu.wpi.first.units.Current
+import edu.wpi.first.units.MutableMeasure
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.Voltage
+import org.team9432.annotation.Logged
+
+interface ClimbIO {
+    val inputs: LoggedClimbInputs
+
+    fun setPower(power:Double) {}
+
+    fun openStopper() {}
+
+    fun closeStopper() {}
+
+    fun disableStopper() {}
```

Comment:
It's not clear what "disable" means.

Reply:
It disables the motor so it doesn't stall. Maybe disable stopper motor is better but I think this is fine.

Reply:
Maybe "deactivate" or "disengage" (I think "disengage" is better)? "disable" sounds like it's going not to work anymore.

Reply:
I don't think it really matters all that much, different people will understand it differently. I think when it comes to naming variables or functions they do need to be indicative but good enough is good enough.

Reply:
Well, I needed to look at the implementation to understand what it means. That means the name is not indicative enough. Let's leave it for now